### PR TITLE
Movies Feed Views Design Finished

### PR DESCRIPTION
**Pull request by albertoag128**
## DESCRIPTION
This is a pull request to show to the views have been done and also how. This pr just covers the feed part and not the detail one yet.

## HOW IT WAS IMPLEMENTED
This time, I tried to increment a little bit the design level by creating a gradient for the background feed and also adding to the items a white background lowing the alpha value to 4%. 
I have also added a "logo" to try to emulate a brand app.
I think the result can be something different from superheroes design and still it does not hit as I would like it to do, but I am still missing some of the design concepts to improve this designs

## KEYWORDS
Design, String, Dimensions, Styles, Resources, ConstraintLayout, View, RecyclerView

## SCREENSHOTS
Single item view on the list

single_item_feed_list

Feed list view

feed_list

### CODE REVIEW

File: app/src/main/res/layout/activity_movies_item_list.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        app:layout_constraintTop_toTopOf="parent"
+        />
+
+    <TextView
+        android:textColor="@color/white"
+        android:id="@+id/yearlabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="@dimen/year_title_size"
+        android:layout_marginTop="@dimen/year_margin_top"
```
Comment (josem-perram) [inline app/src/main/res/layout/activity_movies_item_list.xml:38]: should add a common spacing, for instance, spacing_vertical or spacing_horizontal  8dp, 16dp,

File: app/src/main/res/layout/activity_movies_item_list.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        app:layout_constraintStart_toStartOf="@id/film_title_item"
+        tools:text="Year"
+        />
+
+    <TextView
+        android:textColor="@color/yellow"
+        android:id="@+id/year"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="@dimen/year_title_size"
```
Comment (josem-perram) [inline app/src/main/res/layout/activity_movies_item_list.xml:49]: Should create a commons size with text, for example: headline (22sp)... please, check this [website](https://m2.material.io/design/typography/the-type-system.html#type-scale):

**Review by josem-perram — APPROVED** [review]
Please, check the comments :)

File: app/src/main/res/layout/activity_movies_item_list.xml (hunk 2 above)
Comment (albertoag128) [reply app/src/main/res/layout/activity_movies_item_list.xml:49]: Alright, I will be more carefull next time designing the views for the project and using commons spaces as well as different font sizes for the potential different types of text in the App. Thank you for the feedback :D

**Pull request merged**